### ISSUE TITLE
RE: Я до сих пор не уверен в необходимости воду нагревать

### ISSUE THREAD

**Issue by vvzvlad** [sevprostor/ofwm#1]
>>Я до сих пор не уверен в необходимости воду нагревать и делаю упор на кинетическию обработку.
в современных порошках энзимы и им хочется градусов 30-40 для нормальной работы

**Comment by sevprostor** [comment]
> > > в современных порошках энзимы и им хочется градусов 30-40 для нормальной работы

Да на самом деле мне просто лень было прилеплять туда еще 1 измерительный мост и в очередной раз вскрывать корпус, чтобы вытянуть очередные провода из косы)) Конечно, подогрев лишним не будет. Но все-таки я бы (лично для себя) предпочел 2й клапан воды и горячую трубу. И на то, и на другое еще вполне хватает ресурсов.

Однако же меня тянет больше во всякий оффгрид, когда нет стабильного электропитания, водопровода (надо бак для воды и 2й насос), а вместо современного порошка с энзимами - стратегические запасы хозмыла 72%, либо вообще ничего. И действительно, я сам прихожу к выводу, что при достаточном времени воздействия на тряпки и при достаточном количестве смен вод, порошок не гораздо-то и нужен. С другой стороны, отсутствие порошка можно было бы как раз компенсировать нагревом воды, ага. Надо об этом подумать.

**Comment by vvzvlad** [comment]
Не, без порошка хорошо стираются только что-нибудь типа глины/грязи. Жирные пятна уже не отстирываются, у них слишком высокая энергия связей. Цветные пятна без источника активного кислорода тоже не пропадут. Но в целом темные ткани без загрязнений типа масла/жира стирать без порошка можно, да.

**Comment by sevprostor** [comment]
Да-да, я не спорю и повторяю, что причиной всему лень)) Вообще я стремлюсь к тому, чтобы любую стирку тянул источник мощностью от 800Вт. Это самый слабосильный чемоданчиковый инверторный генератор, имеющийся в продаже. ТЭН я в любом случае собираюсь питать через диммер мотора и включать его в промежутках между масланий. Это важно для генератора - поддерживать более-менее плавную кривую мощности в сети, чтоб он не спотыкался.

ЗЫ: вот кстати надо проверить, как маленький генератор отнесется к актуальному способу диммирования, не заедут ли у него шарики за ролики))

**Comment by sevprostor** [comment]
Дада, отличная мысль кстати. При работе в маломощной сети, чувтсвительной к резким перепадам мощности, перед пуском мотора заранее пригружать сеть спокойным тэном, не создающим никаких лишних возмущений, а потом быстренько переключаться на мотор.
